scheduler/src/cook/mesos/api.clj

```diff
@@ -173,14 +173,15 @@
    ;; length at most 128.
    :name (s/both s/Str (s/pred #(re-matches #"[\.a-zA-Z0-9_-]{0,128}" %) 'under-128-characters-and-alphanum?))
    :priority (s/both s/Int (s/pred #(<= 0 % 100) 'between-0-and-100))
-   :max-retries NonNegInt
+   :max-retries PosInt
```

Comment:
Should there be a unit tests to validate that passing 0 or negative fails the schema validation?